Mlem/Models/Trackers/Post Tracker.swift

```diff
@@ -7,9 +7,14 @@
 
 import Foundation
 import SwiftUI
+import Nuke
 
 class PostTracker: FeedTracker<APIPostView> {
 
+    private let prefetcher = ImagePrefetcher(pipeline: ImagePipeline.shared,
+                                             destination: .memoryCache,
+                                             maxConcurrentRequestCount: 40)
```

Comment:
the libraries default here is `2`, is `40` perhaps overkill?

Reply:
Indeed, but it’s very very carefully tuned, without that the feed gets jumpy as the network catches up 

But with preloading so much in advance and in parallel it’s smooth as butter